cmd/scan/scan.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	_ "log"
```

Comment:
why import this way?

Reply:
i used log for debug and forget delete it